src/core.ts

```diff
@@ -104,7 +104,7 @@ export default {
     Top: '0px -2px 8px rgba(106, 115, 129, 0.12)',
     Left: '-1px 2px 8px rgba(106, 115, 129, 0.22)',
     Right: '1px 0px 8px rgba(106, 115, 129, 0.22)',
-    Focus: '0px 0px 0px 2px rgba(82, 138, 224, 0.8)',
+    Focus: '0px 0px 0px 2px rgba(45, 100, 239, 1)',
```

Comment:
Shouldn't the opacity be 75%? I'm just reading from the attached issue.

Also if I convert `#095ADB` to rgb, I get `rgb(9, 90, 219)` and the rgba value would be `rgb(9, 90, 219, 0.75)`.